premake5.lua

```diff
@@ -79,7 +79,9 @@ filter("configurations:Release")
     "NDEBUG",
     "_NO_DEBUG_HEAP=1",
   })
-  optimize("On")
+  optimize("speed")
+  inlining ("Auto")
+  floatingpoint ("Fast")
```

Comment:
Does changing this to Fast break anything?

Reply:
Not that I've noticed. It does seem to increase the problem with audio. However, increasing the audio thread priority resolves that.